### PR TITLE
[solidjs-pod] Add content for Courses

### LINKED ISSUE #295: [solidjs-pod] Add content for Courses
# Add Courses content for framework

## Background

As a visitor of the framework specific Framework.dev site I would like to have the ability to see enough Courses content to keep me informed and find this site to be resourceful and useful.

## Acceptance

- [x] Research a list of Courses
- [x] Add the Courses information in the data files
- [x] Test each individual look and external link

NOTE:
- This task is timeboxed to a maximum of 4 hours.

### PR DESCRIPTION
**Pull request by kodejuice**
Closes #295 

## Type of change

- [x] Content addition
- [ ] Bug fix
- [ ] Behavior change

## Summary of change

Add courses for SolidJS

### CODE REVIEW

File: packages/site/src/data/solidjs/courses.ts [hunk 1]
```diff
@@ -4,19 +4,141 @@ export const courseTags = [
...
+			"lifecycle methods",
+			"state management",
+			"events",
+			"effects",
+			"conditionals",
+			"loops",
+		],
+	},
+	{
+		title: "Lets Learn Solid JS",
```
Comment (vyktoremario) [inline packages/site/src/data/solidjs/courses.ts:68]: "Let's ..."

File: packages/site/src/data/solidjs/courses.ts [hunk 2]
```diff
@@ -4,19 +4,141 @@ export const courseTags = [
...
+		description:
+			"In this tutorial, we’ll explore how SolidJS works. To do this, we’ll build a sample “to do” app to showcase what this great framework has to offer.",
+		paymentType: "free",
+		level: "intermediate",
+		format: "text",
+		href: "https://blog.logrocket.com/introduction-solidjs/",
+		tags: ["state management", "routing", "data fetching"],
+	},
+	{
+		title: "Lets Learn SolidJS",
```
Comment (vyktoremario) [inline packages/site/src/data/solidjs/courses.ts:92]: Same as above

Comment (vyktoremario) [inline packages/site/src/data/solidjs/courses.ts:92]: Same as above

**Review by vyktoremario — APPROVED** [review]
LGTM
Is there a specific amount of courses you are expected to provide or this is the amount you could find?

**Comment by kodejuice** [comment]
> LGTM Is there a specific amount of courses you are expected to provide or this is the amount you could find?

There're not much courses on solidJS on the web but these ones are sufficient i would say

**Review by hdJerry — APPROVED** [review]

**Pull request merged**